sos/report/plugins/dnf.py

```diff
@@ -55,7 +55,13 @@ def setup(self):
             '/etc/dnf/modules.d/.*.module': 'dnf_modules'
         })
 
-        self.add_copy_spec("/etc/dnf/")
+        self.add_copy_spec([
+            "/etc/dnf/",
+            "/etc/dnf/vars/",
```

Comment:
This should already be collected by the previous copyspec - unless we're hitting size limiting somewhere?

Reply:
Ah, I was blind.. thanks for pointing to the dup.